packages/xrpl/HISTORY.md

```diff
@@ -1,207 +1,244 @@
 # xrpl.js (ripple-lib) Release History
 
 Subscribe to [the **xrpl-announce** mailing list](https://groups.google.com/g/xrpl-announce) for release announcements. We recommend that xrpl.js (ripple-lib) users stay up-to-date with the latest stable release.
+
```

Comment:
These HISTORY changes are unnecessary. If you want them, they should be in a separate PR, because otherwise they make this PR look massive.